exchanges/execute/src/execute.ts

```diff
@@ -0,0 +1,60 @@
+import { pipe, share, filter, map, fromPromise, mergeMap, merge } from 'wonka';
+import {
+  GraphQLSchema,
+  GraphQLFieldResolver,
+  GraphQLTypeResolver,
+  execute,
+} from 'graphql';
+import { Exchange, makeResult, makeErrorResult } from '@urql/core';
+
+interface ExecuteExchangeArgs {
+  schema: GraphQLSchema;
+  rootValue?: any;
+  contextValue?: any;
+  fieldResolver?: GraphQLFieldResolver<any, any>;
+  typeResolver?: GraphQLTypeResolver<any, any>;
+}
+/** Exchange for executing queries locally on a schema using graphql-js. */
+export const executeExchange = ({
+  schema,
+  rootValue,
+  contextValue,
+  fieldResolver,
+  typeResolver,
+}: ExecuteExchangeArgs): Exchange => ({ forward }) => {
+  return ops$ => {
+    const sharedOps$ = share(ops$);
+    const targetOperationTypes = ['query', 'mutation', 'subscription'];
```

Comment:
I don’t think you’d want subscriptions here because this is missing the necessary support stuff from `graphql-subscriptions`. Let’s maybe keep it to queries and mutations for now? 🤔

Reply:
Looks like there's [no combined function yet](https://github.com/graphql/graphql-js/issues/2144) either - maybe something worth looking at in the future.

```suggestion
    const targetOperationTypes = ['query', 'mutation'];
```